makiflow/layers/untrainable_layers.py

```diff
@@ -591,6 +640,8 @@ class ActivationLayer(SimpleForwardLayer):
     TYPE = 'ActivationLayer'
     ACTIVATION = 'activation'
 
+    _ERROR_ACTIVATION_INPUT_NONE = "Activation can't None"
```

Comment:
The same as previous. Better to use the word 'exception'. For example, `EXCEPTION_ACTIVATION_IS_NONE`.
Fix the description of the exception -> "Activation can't be None".